kornia/feature/disk/disk.py

```diff
@@ -0,0 +1,121 @@
+from typing import List, Optional, Tuple
+
+import torch
+
+from kornia.core import Tensor
+from kornia.utils.helpers import map_location_to_cpu
+
+from .detector import heatmap_to_keypoints
+from .structs import DISKFeatures
+from .unets import Unet
+
+
+class DISK(torch.nn.Module):
```

Comment:
`from kornia.core import Module`
```suggestion
class DISK(Module):
```

Reply:
What is the difference?

Reply:
We are exposing some things on the `core' to facilitate manteability and try to do something in the future